Makefile

```diff
@@ -98,13 +94,25 @@ vendor:
 	cp -rf --no-preserve=mode "$${_src}" "$${_dst}"
 .PHONY: vendor
 
+# GENERATE_YAML_ENV_VARS is the list of all environment variables expected by generate-yaml.sh
+# to build example manifests for clusterctl. The list of environment variables here should contain
+# the environment variables expected in generate-yaml.sh
+GENERATE_YAML_ENV_VARS = -e CLUSTER_NAME -e SERVICE_CIDR -e CLUSTER_CIDR -e CAPV_YAML_VALIDATION \
+		      -e VSPHERE_USER -e VSPHERE_PASSWORD -e VSPHERE_SERVER -e VSPHERE_DATACENTER \
+		      -e VSPHERE_DATASTORE -e VSPHERE_NETWORK -e VSPHERE_RESOURCE_POOL -e VSPHERE_FOLDER \
+		      -e VSPHERE_TEMPLATE -e VSPHERE_DISK -e VSPHERE_DISK_SIZE_GB -e CAPV_MANAGER_IMAGE \
+		      -e KUBERNETES_VERSION
+
 ####################################
 # DEVELOPMENT Build and Push targets
 ####################################
 
 # Create YAML file for deployment
-dev-yaml: | $(KUSTOMIZE)
-	CAPV_MANAGER_IMAGE=${DEV_IMG} cmd/clusterctl/examples/vsphere/generate-yaml.sh
+dev-yaml: clusterctl-tools
+	docker run -v $(PWD):/go/src/sigs.k8s.io/cluster-api-provider-vsphere \
```

Comment:
Hi @andrewsykim,

I don't know that we should assume `PWD` is always set. At the top of the Makefile please consider adding:

```makefile
PWD := $(abspath .)
```